content/en/blog/2022/2022-03-15-k8s-triage-bot-helper-ci-job/K8s Triage bot Helper CI Job_ ‘make update’.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: blog
+title: >-
+    "K8s Triage bot Helper CI Job: ‘*make update*’"
```

Comment:
```suggestion
    Kubernetes Build Helper CI job: Run "make update" within CI
```

Reply:
I was changing those, I accidentally deleted the last changes when my pc went down. I shall push them soon.